geometry_msgs/Point start
geometry_msgs/Point goal
int32 length
int32 width
int32 n_pts_per_dir
---
trajectory_msgs/MultiDOFJointTrajectory multiDOFtrajectory
int32 unknown
bool path_found
